integration/spark/spark3/src/main/java/io/openlineage/spark3/agent/lifecycle/plan/catalog/AbstractDatabricksHandler.java

```diff
@@ -0,0 +1,101 @@
+/*
+/* Copyright 2018-2022 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.spark3.agent.lifecycle.plan.catalog;
+
+import io.openlineage.spark.agent.util.DatasetIdentifier;
+import io.openlineage.spark.agent.util.PathUtils;
+import io.openlineage.spark.api.OpenLineageContext;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.reflect.MethodUtils;
+import org.apache.hadoop.fs.Path;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.catalyst.TableIdentifier;
+import org.apache.spark.sql.connector.catalog.Identifier;
+import org.apache.spark.sql.connector.catalog.TableCatalog;
+import scala.Option;
+
+/**
+ * The AbstractDatabricksHandler is intended to support Databricks' custom proxies which supersede
+ * the open source Delta Catalog. The proprietary class name of
+ * com.databricks.sql.transaction.tahoe.catalog.DeltaCatalog is used by Databricks instead of the
+ * open source class name of org.apache.spark.sql.delta.catalog.DeltaCatalog. It is used in the same
+ * way as the {@link DeltaHandler}.
+ */
+@Slf4j
+public abstract class AbstractDatabricksHandler implements CatalogHandler {
+  final String DATABRICKS_CLASS_NAME_STRING;
```

Comment:
Uppercase with underscore is a naming convention for constants in Java, while `DATABRICKS_CLASS_NAME_STRING` is not such. Consider renaming it to `databricksClassNameString`.

Reply:
Changes made!  Thank you so much, for the excellent guidance as usual, Pawel!